opm/parser/eclipse/Deck/DeckRecord.cpp

```diff
@@ -29,6 +29,10 @@
 namespace Opm {
 
 
+    DeckRecord::DeckRecord( size_t size ) {
```

Comment:
It certainly makes a lot of sense to reserve the size of each individual `DeckItem` - but reserving the number of `DeckItem` instances in a `DeckRecord` probably does not make much sense performance wise?